editor/editor_interface.h

```diff
@@ -145,6 +148,7 @@ class EditorInterface : public Object {
 	void popup_property_selector(Object *p_object, const Callable &p_callback, const PackedInt32Array &p_type_filter = PackedInt32Array(), const String &p_current_value = String());
 	void popup_method_selector(Object *p_object, const Callable &p_callback, const String &p_current_value = String());
 	void popup_quick_open(const Callable &p_callback, const TypedArray<StringName> &p_base_types = TypedArray<StringName>());
+	void popup_create_dialog(const Callable &p_callback, const StringName &p_base_type = "", const TypedArray<StringName> &p_custom_type_blacklist = TypedArray<String>(), const Dictionary &p_custom_suffix = Dictionary());
```

Comment:
The Dictionary could be typed.
(AFAIK you ran into some problems with it; it would still be nice to resolve it though)

Reply:
Well the problem is that adding type technically breaks compatibility, so we should do it swiftly.